.github/workflows/locale.yml

```
@@ -0,0 +1,98 @@
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get postgres version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:26: Double quote to prevent globbing and word splitting

(shellcheck)